docs/source/architecture.rst

```diff
@@ -276,11 +393,10 @@ View changing mode has the following steps:
    operation.
 
 The next primary node is determined by the node ID, in sequential order, based
-based on the order of nodes in the ``sawtooth.consensus.pbft.peers`` on-chain
-setting. The first node on the list has ID 0 and is the first primary for view
-0; the next node has ID 1 and is the second primary for view 1; and so on.
-After all nodes on the list have been the primary, the algorithm returns to
-node 0.
+on the order of nodes in the ``sawtooth.consensus.pbft.peers`` on-chain setting.
+The first node on the list has ID 0 and is the first primary for view 0; the
```

Comment:
"ID 0", "ID 1", etc. is outdated information; this was from back in the beginning where nodes had numbered ID's. Now, the only concept of a node ID is the node's public key. This paragraph should be updated to reflect that and match what's in the "View Changes: Choosing a New Primary" section

Reply:
In order to avoid duplicated text, I'll replace this info with a link to the (correct) info in "View Changes: Choosing a New Primary".